tidy3d/plugins/smatrix/ports/rectangular_lumped.py

```diff
@@ -194,9 +194,12 @@ def compute_current(self, sim_data: SimulationData) -> FreqDataArray:
         inject_center = h_cap_coords_along_injection[orth_index]
         # Some sanity checks, tangent H field coordinates should be directly above
         # and below the coordinates of the resistive sheet
-        assert orth_index > 0
-        assert inject_center < h_coords_along_injection[orth_index]
-        assert h_coords_along_injection[orth_index - 1] < inject_center
+        if orth_index <= 0:
+            raise AssertionError
+        if inject_center >= h_coords_along_injection[orth_index]:
+            raise AssertionError
+        if h_coords_along_injection[orth_index - 1] >= inject_center:
+            raise AssertionError
```

Comment:
@dmarek-flex If you have suggestions for proper error messages to raise here that'd be great.

Reply:
Awesome, thanks!